app/models/user.rb

```diff
@@ -24,7 +24,10 @@ class User < ApplicationRecord
   has_many :phylo_trees
 
   validates :email, presence: true
-  validates :name, presence: true, format: { with: /\A[a-zA-Z -]+\z/, message: "only allows letters" }
+  validates :name, presence: true, format: {
+    # See https://www.ascii-code.com/
+    with: /\A[- 'a-zA-ZÀ-ÖØ-öø-ÿ]+\z/, message: "Name must contain only letters, apostrophes, dashes or spaces"
```

Comment:
What was the rationale for picking these characters in particular? (are they the most popular?) I notice there are some additional Latin characters in www.ascii-code.com, such as 131 "Latin small letter f with hook", but I guess they aren't compatible with UTF-8. Figuring out a good regex for international names seems like a rabbit hole, but it would be good to add a comment explaining why you choose these characters.

Reply:
Good question. I added this comment

```    # See https://www.ascii-code.com/. These were the ranges that captured the
    # common accented chars I knew from experience, leaving out pure symbols.
```